### DISCUSSION TITLE
Include Child Issues (Sub-Tasks) in Issue views (like Linked Issues)

### DISCUSSION THREAD

**Discussion by metafeather** [ankitpokhrel/jira-cli#338] [Ideas] [answered]
Include Child Issues (Sub-Tasks) in Issue views (like Linked Issues).

These can be in their own section, I'd suggest before Linked Issues since they are more directly related.

**Comment by ankitpokhrel** [comment, marked as answer]
@metafeather This feature is available since the latest beta release [v1.0.0-beta.2](https://github.com/ankitpokhrel/jira-cli/releases/tag/v1.0.0-beta.2)
